src/main/java/uk/gov/hmcts/darts/arm/component/impl/ArmRetentionEventDateCalculatorImpl.java

```diff
@@ -56,19 +56,21 @@ public boolean calculateRetentionEventDate(Integer externalObjectDirectoryId) {
                         Integer confidenceScore = confidenceAware.getRetConfScore();
                         String confidenceReason = confidenceAware.getRetConfReason();
 
-                        UpdateMetadataResponse updateMetadataResponseMedia = armDataManagementApi.updateMetadata(
-                            externalObjectDirectory.getExternalRecordId(), armRetentionDate, confidenceScore, confidenceReason);
+                        if (confidenceScore != null) {
+                            UpdateMetadataResponse updateMetadataResponseMedia = armDataManagementApi.updateMetadata(
```

Comment:
@hemantasharma1129 is this ok or should we always push the null score through?

Reply:
@mestebanez As discussed we can add null and 0 check for score. Though both score and reason should get calculated along with Retention date so score wont be null.